### PR TITLE
LG-4065:  Spacing[900]

### PR DESCRIPTION
**Pull request by bruugey**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-4065](https://jira.mongodb.org/browse/LG-4065)

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

File: packages/tokens/src/spacing.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ const spacing = {
   400: 16,
   600: 24,
   800: 32,
+  900: 36,
```
Comment (TheSonOfThomp) [inline packages/tokens/src/spacing.ts:12]: Noticed this after I made the ticket, but I think we also need 1000 and 1400

**Review by TheSonOfThomp — APPROVED** [review]

File: packages/tokens/src/spacing.ts (hunk 1 above)
Comment (bruugey) [reply packages/tokens/src/spacing.ts:12]: Added!

**Review by stephl3 — APPROVED** [review]

File: packages/tokens/src/spacing.ts [hunk 2]
```diff
@@ -9,7 +9,10 @@ const spacing = {
...
+  1000: 40,
   1200: 48,
+  1400: 46,
```
Comment (TheSonOfThomp) [inline packages/tokens/src/spacing.ts:15]: should be 56

Comment (bruugey) [reply packages/tokens/src/spacing.ts:15]: ugh typo thank you

**Review by TheSonOfThomp — APPROVED** [review]
LGTM!

**Pull request merged**